python/README.md

```diff
@@ -0,0 +1,141 @@
+# The Splunk Examples for Python SDK
+
+The Splunk Examples for Python SDK contains examples that are designed to enable developers to analyse and run the various examples for python SDK.
+
+The Splunk platform is a search engine and analytic environment that uses a distributed map-reduce architecture to efficiently index, search, and process large time-varying data sets.
+
+The Splunk platform is popular with system administrators for aggregation and monitoring of IT machine data, security, compliance, and a wide variety of other scenarios that share a requirement to efficiently index, search, analyze, and generate real-time notifications from large volumes of time-series data.
+
+The Splunk developer platform enables developers to take advantage of the same technology used by the Splunk platform to build exciting new applications.
+
+## Getting started with the Splunk Examples for Python SDK
+
+### Requirements
+
+* Python 3.7
+  
+  The Splunk Enterprise SDK for Python has been tested with and v3.7.
+
+* Splunk Enterprise
+
+  If you haven't already installed Splunk Enterprise, download it [here](http://www.splunk.com/download). 
+  For more information, see the Splunk Enterprise [_Installation Manual_](https://docs.splunk.com/Documentation/Splunk/latest/Installation).
+
+### Install the SDK
+
+Use the following commands to install the Splunk Enterprise SDK for Python libraries.
+
+Use `pip`:
+
+    [sudo] pip install splunk-sdk
+
+## Testing Quickstart
+
+You'll need `docker` and `docker-compose` to get up and running using this method.
+
+```
+make up SPLUNK_VERSION=8.0
```

Comment:
Minor comment, can we change this to `make up SPLUNK_VERSION=latest`? That way this line won't need updating in the future

Reply:
updated